dlt/destinations/redshift/redshift.py

```diff
@@ -108,6 +108,8 @@ def execute(self, table: TTableSchema, bucket_path: str) -> None:
         file_type = ""
         dateformat = ""
         compression = ""
+        if table_schema_has_type(table, "time"):
```

Comment:
heh. so let it be. please improve the message below. ie. 
"Redshift cannot load TIME columns from {ext} files. Switch to direct INSERT file format or convert datetime.time objects in your data to str or datetimes"

please also add a note in Redshift docs (we have other exceptions there)

Reply:
Ok, blocked this in both athena and redshift and added to tests. Now we wait for CI :pray: